src/dot-components/Wishlist/wishlist.css

```diff
@@ -0,0 +1,56 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+
+body{
+    font-family: "Montserrat", sans-serif;
+}
+.wishlist-container{
+    gap: 40px;
```

Comment:
better to use relative units

Reply:
Oh yeah, forgot to convert it. Will do that.